projects/libs/core-library-typescript/test/consensus/block-builder.spec.ts

```diff
@@ -26,32 +28,23 @@ function aGenesisBlock(): types.Block {
   });
 }
 
-function aDummyTransaction(addressId): types.Transaction {
-  const senderAddress: types.UniversalAddress = {
-    id: new Buffer(addressId),
-    scheme: 0,
-    checksum: 0,
-    networkId: 0
-  };
-
+function aDummyTransaction(publicKey: Buffer): types.Transaction {
```

Comment:
isn't this duplicated from the testkit? (`dummy-transaction.ts`)? 
anyway, I'd refactor both to a file `transaction-builders` which also builds the transaction set.